### ISSUE TITLE
Feature: introduce a more accessible way to start a new conversation in the playgrounds

### ISSUE THREAD

**Issue by massi-ang** [aws-samples/aws-genai-llm-chatbot#283]
Currently the chatbot allows to start new conversation by opening the right side panel and clicking on "New Session". This is fine, but having a mechanism (like a button) in the main UI would make it easier to start new conversations.

**Comment by Rob-Powell** [comment]
I agree this is a good feature to have but not a replacement for #306 clicking this button every time if you know each line entered is to be unique is simply redundant. @bigadsoleiman
